pixeltable/func/udf.py

```diff
@@ -82,6 +86,9 @@ def make_function(
         function_path = f'{decorated_fn.__module__}.{decorated_fn.__qualname__}'
     else:
         function_path = None
+    if is_stored:
+        # force storing the function in the db
+        function_path = None
```

Comment:
This would be clearer to me:

```
if force_stored:
    function_path = None
elif decorated_fn.__module__ != '__main__' and decorated_fn.__name__.isidentifier():
    funciton_path = ...
else:
    function_path = None
```